lib/jackal-kitchen/tester.rb

```diff
@@ -200,6 +202,29 @@ def insert_kitchen_ssh(path)
         end
       end
 
+      def update_spec_helpers(path)
+        spec_helpers = []
+        Find.find(path) do |location|
+          spec_helpers << location if location =~ /spec_helper.rb/
+        end
+        spec_helpers.each do |spec_helper|
+          content = File.readlines(spec_helper)
+          open(spec_helper, 'a') { |f|
+            f << "RSpec.configure do |config|\n"
```

Comment:
oh ya, I kept the syntax similar to what was already in the file, but maybe we should go all heredoc all the way? thoughts @ckuttruff? @cwjohnston